pset-iterators.js

```diff
@@ -43,6 +43,11 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+//  find the first president that was a member of the party named Whig
+const firstWhigPresident = presidents.find(selectedPresidentObj => selectedPresidentObj.party === 'Whig')
+// print to console
+console.log(firstWhigPresident.president)
+
```

Comment:
@sad3xoxo Great work here on this solution using the find method to return the first president of the whig party.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L35)